Gemfile

```diff
@@ -0,0 +1,2 @@
+source 'https://rubygems.org'
+gem 'json-api-vanilla', '~> 1.0.1'
```

Comment:
Just use the `gemspec` keyword in your gemfile instead of double specifying your dependencies.